kong/clustering/version_negotiation/init.lua

```diff
@@ -354,4 +354,40 @@ function _M.get_negotiated_service(name)
   return val.version, val.message
 end
 
+--- detect '/v1/wrpc' endpoint
+--- if there is no '/v1/wrpc', fallback to websocket + json
+function _M.head_request_version(conf, cert, cert_key)
```

Comment:
Could we name it like `check_wrpc_protocol_support`?

Reply:
Good idea, but at first we must pass the ci test.